homework/taniab/tania_hw_6.py

```diff
@@ -0,0 +1,31 @@
+from functools import reduce
+
+
+def count_holes(num):
+    holes_dict = {
+        0: 1,
```

Comment:
I think you can use str obj as key, so then you don't need change number type from str to int. [18] line